### ISSUE TITLE
Port mapping issue in docker container

### ISSUE THREAD

**Issue by veloce** [ceph/cn#116]
I installed the latest version v2.3.0 from the binary. Installation went fine, but after creating a cluster it is not possible to access it from the host.
The command `cn cluster status my-cluster` times out.

The reason is the wrong port mapping in docker:
```
$ docker ps
NAMES         PORTS                                                                   STATUS
ceph-nano-my-cluster   0.0.0.0:5000->5000/tcp, 0.0.0.0:8000->8000/tcp   Up 10 minutes
```

Cluster service is running on 8080 inside the container, so it can never be reached from the host.
I changed the port mapping configuration on the docker container like so `0.0.0.0:8000->8080/tcp` and it solved the problem.

**Comment by deep-j** [comment]
Can you provide the container log and details of the ceph image used?
Thanks.

**Comment by leseb** [comment]
Closed via #118
